### PR TITLE
Add/update resource schemas with schemas generated from swagger docs

### PR DESCRIPTION
**Pull request by daschult**
I have created a tool that allows us to generate resource schemas from swagger docs (https://github.com/Azure/autorest/pull/1018). This pull request adds/updates the schemas that I generated during my tests of my generation tool.

### CODE REVIEW

File: schemas/2015-01-01/deploymentTemplate.json [hunk 1]
```diff
@@ -84,7 +81,60 @@
...
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Network.json#/resourceDefinitions/networkSecurityGroups_securityRules"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Network.json#/resourceDefinitions/publicIPAddresses"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Network.json#/resourceDefinitions/routeTables"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Network.json#/resourceDefinitions/routeTables_routes"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Network.json#/resourceDefinitions/virtualnetworkgateways"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Network.json#/resourceDefinitions/virtualnetworks"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Network.json#/resourceDefinitions/virtualnetworks"},
```
Comment (wenwu449) [inline schemas/2015-01-01/deploymentTemplate.json]: duplication of virutalNetworks

Comment (daschult) [inline schemas/2015-01-01/deploymentTemplate.json]: Good catch. Removed.

**Comment by kuangweizhang** [comment]
@daschult LGTM for CDN schema, can you please also generate the schema from this swagger(https://github.com/Azure/azure-rest-api-specs/tree/master/arm-cdn/2015-06-01/swagger) as well?

**Comment by daschult** [comment]
Is this pull request waiting on anything?

**Pull request closed without merging**